src/patternfly/components/AlertGroup/docs/alert-group.md

```diff
@@ -6,7 +6,6 @@
 
 | Attribute | Applied To | Outcome |
 | -- | -- | -- |
-| `role="status"` | `.pf-c-alert-group` |  Communicates contents of the alert message to the user. **Required** |
```

Comment:
Can you remove this whole a11y section?

<img width="1253" alt="Screen Shot 2019-03-13 at 1 54 37 PM" src="https://user-images.githubusercontent.com/35148959/54306555-98f3bf00-4597-11e9-91b7-8cdbf5b87021.png">

Reply:
Yep